xml/admin_monitoring_alerting.xml

```diff
@@ -133,39 +97,19 @@
 
  <sect1 xml:id="prometheus">
    <title>&prometheus;</title>
-  <para>The exporter based configuration that can be passed through the pillar.
-    These groups map to exporters that provide data. The node exporter is
-    present on all nodes, &ceph; is exported by the &mgr; nodes, &prometheus;
-    and &grafana; is exported by the respective &prometheus; and &grafana; nodes.</para>
+  <para>
+    <!-- Add intro about prometheus
+  https://docs.ceph.com/docs/master/mgr/prometheus/ (see below for variable list)-->
+    The node exporter is present on all nodes. &ceph; data is exported by
+    the &prometheus; &mgr; module.</para>
```

Comment:
Thinking about it, I think the prometheus manager module to be enabled by the user. This is done by issuing `ceph mgr module enable prometheus`.

Reply:
This is a step that the user needs to execute to be able to get from the prometheus manager module. It needs to be run before Prometheus is deployed. If it was not run when prometheus was deployed, the user needs to redeploy prometheus like so `ceph orch redeploy prometheus`.